includes/admin/class-wcmp-export.php

```diff
@@ -823,55 +826,121 @@ public function saveShipmentData(WC_Order $order, array $shipment): void
     }
 
     /**
-     * TODO: For MyParcel NL, currently not necessary for BE.
+     * @param array  $order
+     * @param string $shipping_method_id
      *
+     * @return bool|int
+     * @throws \Exception
+     */
+    public function getOrderShippingClass($order, $shipping_method_id = '')
```

Comment:
shipping_method_id in camelCase, net als andere variables hier.

Reply:
Done